dev/Deployment/DeploymentManager.cpp

```diff
@@ -59,30 +66,43 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppSDK::implementa
             }
         }
 
-        auto status{ winrt::make<implementation::DeploymentStatus>(SUCCEEDED(verifyResult), FAILED(verifyResult), verifyResult) };
-        return status;
-    }
+        DeploymentStatus status{};
+        if (SUCCEEDED(verifyResult))
+        {
+            status = DeploymentStatus::Ok;
+        }
+        else
+        {
+            status = DeploymentStatus::PackageInstallRequired;
```

Comment:
Feels like this should be a more precise mapping? Are all "fails" implying install required? Or can a fail like "no internet" be made more precise? Probably OK for now.

Reply:
For checking status currently, yes a failure to find the registered package implies a package install is required. I expect we will need additional status types later on, such as PackageInstallPending or PackageInstallInProgress and similar results if the package is missing and we are in the process of installing it (such as a web request).